### PR TITLE
[WIP] bypass OverflowError when calculate perplexity 

### PR DESCRIPTION
**Pull request by mrchypark**
I got OverflowError when training.
So i fixed.

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -197,14 +197,15 @@ def trainEpoch(epoch):
...
-                print("Epoch %2d, %5d/%5d batches; perplexity: %6.2f; %3.0f Source tokens/s; %6.0f s elapsed" %
-                      (epoch, i, len(trainData),
-                      math.exp(report_loss / report_words),
-                      report_src_words/(time.time()-start),
-                      time.time()-start_time))
-
-                report_loss = report_words = report_src_words = 0
-                start = time.time()
+                try:
+                  ans = math.exp(report_loss / report_words)
```
Comment (guillaumekln) [inline train.py]: Could you use a more meaningful variable name (like `perplexity`)? Also please use the same indentation as the rest of the project (4 spaces).

Comment (guillaumekln) [inline train.py]: Could you use a more meaningful variable name (like `perplexity`)?

File: train.py [hunk 2]
```diff
@@ -197,14 +197,15 @@ def trainEpoch(epoch):
...
+                except OverflowError:
+                  ans = float('inf') 
+                print("Epoch %2d, %10d/%10d batches; perplexity: %6.2f; %3.0f Source tokens/s" %
```
Comment (guillaumekln) [inline train.py]: Why did you drop the `%6.0f s elapsed` section of the log?

File: train.py [hunk 3]
```diff
@@ -197,14 +197,15 @@ def trainEpoch(epoch):
...
+                  (epoch, i, len(trainData), ans,
```
Comment (guillaumekln) [inline train.py]: Indentation is off: it should be aligned with the quote of the line just above.

File: train.py [hunk 4]
```diff
@@ -197,14 +197,15 @@ def trainEpoch(epoch):
...
+                  report_src_words/(time.time()-start)))
+                  report_loss = report_words = report_src_words = 0
```
Comment (guillaumekln) [inline train.py]: Indentation is off: it should be aligned with the `print` statement.

File: train.py (hunk 1 above)
Comment (mrchypark) [inline train.py]: Amazing thank you for feedback!

File: train.py (hunk 2 above)
Comment (mrchypark) [inline train.py]: I should be mistaken.

File: train.py (hunk 3 above)
Comment (mrchypark) [inline train.py]: I will try!

File: train.py (hunk 4 above)
Comment (mrchypark) [inline train.py]: Thank you again.

**Pull request closed without merging**